packages/esm-patient-medications-app/src/order-basket/order-basket-item.scss

```diff
@@ -1,21 +1,49 @@
 @import "../root";
 
+.clickableTileDesktop {
+  margin-bottom: 0.5rem;
+}
+
+.clickableTileTablet {
+  &:not(:last-of-type) {
+    border-bottom: 0.063rem solid $ui-03;
```

Comment:
0.063rem? I mean are we going to moon here;)? Where is this very specific number coming from. Why not 0.065 or so?

Reply:
😄 just strict (maybe too overly so) conformance to the measurements outlined in the designs https://app.zeplin.io/project/60d59321e8100b0324762e05/screen/60d5e127606309ba4cfdafea. 

<img width="324" alt="Screenshot 2021-09-12 at 13 03 26" src="https://user-images.githubusercontent.com/8509731/132983504-9e888f38-d043-4895-8185-b373610356c7.png">

`1px bottom border` ~= 0.0.625rem

Would you prefer to have these really small measurements in px than rem?

Reply:
If you use px or rem depends on if there is a dependency on the primary font size or not. If such odd numbers come out it usually means that there is no dependency. If for instance we have 4px or 8px with a primary font size of 16px then chances are high that 0.25 or 0.5rem are meant, but overall this is a discussion (or specification) with the designer.